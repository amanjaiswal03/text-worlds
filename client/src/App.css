.topbar {
  margin-top: 10vh;
}

.logo-parent {
  display: flex;
  justify-content: center;
  align-items: center;
}

.logo {
  width: 40vw;
}

.menu-button{
  display: flex;
  flex-direction: row;
}

.world-list{
  margin-top: 10vh;
  margin-left: 10vw;
  margin-bottom: 10vw;
}
.add-world{
  margin-top: 10vh;
  margin-left: 10vw;
  margin-right: 10vw;
}


