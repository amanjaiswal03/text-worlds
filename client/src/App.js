import React from 'react';
import './App.css';
import World from './Components/World/World';
import ApolloClient from 'apollo-boost';
import { ApolloProvider } from '@apollo/react-hooks';
import { BrowserRouter, Route } from 'react-router-dom';
import WorldList from './Components/Homepage/WorldList';
import AddWorld from './Components/AddWorld/AddWorld';
import Signup from './Components/Forms/Signup';
import Login from './Components/Forms/Login';

const client = new ApolloClient({
  uri: 'http://localhost:4000/graphql',
});


function App() {
  return (
    <ApolloProvider client={client}>
      <BrowserRouter>
        <Route exact path = '/' component = {WorldList}/>
        <Route exact path = '/add-world' component = {AddWorld} />
        <Route exact path = '/signup' component = {Signup} />
        <Route exact path = '/login' component = {Login} />
        <Route exact path = '/world/:id' component = {World}/>
      </BrowserRouter>
   </ApolloProvider>
  );
}

export default App;
