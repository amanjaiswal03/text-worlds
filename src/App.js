import React from 'react';
import logo from './logo.svg';
import './App.css';
import AccordionExampleStyled from './Accordion/Accordion';

function App() {
  return (
    <div className="App">
      <AccordionExampleStyled/>
    </div>
  );
}

export default App;
