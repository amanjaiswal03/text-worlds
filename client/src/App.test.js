import React from 'react'
import {render, cleanup} from '@testing-library/react'
import App from './App'

// cleanup after each test to avoid memory leaks 
afterEach(cleanup)
 
//testing to ensure fragment of the App component matches the snapshot
 it('should take a snapshot', () => {
    const { asFragment } = render(<App />)
    
    expect(asFragment(<App />)).toMatchSnapshot()
  });