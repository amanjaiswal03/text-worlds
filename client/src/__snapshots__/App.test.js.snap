// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="sc-bdVaJa fJUCAf"
  >
    <div
      class="transition-group"
    >
      <section
        class="route-section"
      >
        <div
          class="ui active centered loader"
        />
      </section>
    </div>
  </div>
</DocumentFragment>
`;
